reporter/otlp_reporter.go

```diff
@@ -81,6 +81,7 @@ type traceEvents struct {
 	mappingEnds        []libpf.Address
 	mappingFileOffsets []uint64
 	timestamps         []uint64 // in nanoseconds
+	pid                uint64
```

Comment:
PID should be added to `traceAndMetaKey` instead of `traceEvents`, to avoid collisions where the same hash/comm shows up across different PIDs.